tests/device_tests/test_msg_getaddress_segwit_native.py

```diff
@@ -78,6 +113,25 @@ def test_show_segwit(client, show_display, coin, path, script_type, address):
     )
 
 
+@pytest.mark.skip_t1
```

Comment:
Let's add a comment this tests against vectors from https://github.com/bitcoin/bips/blob/master/bip-0086.mediawiki#test-vectors

Reply:
https://github.com/trezor/trezor-firmware/pull/1891/commits/fef2ef401b4755fdd0e7b0e435f43ca1c5ea3fa1